src/mmw/apps/modeling/tasks.py

```diff
@@ -120,34 +122,51 @@ def make_gt_service_call_task(model_input):
     """
     Call geotrellis and calculate the tile data for use in the TR55 model.
     """
-    # TODO call the real service
-    # It would be nicer to use HttpDispatchTask but there were import problems.
-    # In testing we could not make it work so we will probably need to default
-    # to urllib2.
-    # return HttpDispatchTask(url="http://gtservice.internal/endpoint",
-    #                         method="POST", input=model_input)
 
-    # TODO remove me. For testing fetch something over the web.
-    result = urllib2.urlopen('http://azavea.com')
-    return result.read()
+    # TODO actually call geotrellis with model_input
+    census = {
+        'cell_count': 8,
+        'distribution': {
+            'c:commercial': {'cell_count': 5},
+            'a:deciduous_forest': {'cell_count': 3}
+        },
+        'modifications': [
+            {
+                'bmp': 'no_till',
+                'cell_count': 1,
+                'distribution': {
+                    'a:deciduous_forest': {'cell_count': 1},
+                }
+            },
+            {
+                'reclassification': 'd:rock',
+                'cell_count': 1,
+                'distribution': {
+                    'a:deciduous_forest': {'cell_count': 1}
+                }
+            }
+        ]
+    }
+    return census
 
 
 @shared_task
-def run_tr55(model_input, landscape):
+def run_tr55(census, model_input):
     """
     A thin Celery wrapper around our TR55 implementation.
     """
-    time.sleep(5)
-
-    # TODO Get the real simpulation data.
-    # (q, et, inf) = simulate_year(landscape)
-
-    # TODO Dummy data. Remove me.
-    q = 2
-    et = 3
-    inf = 4
-    return {
-        'q': q,
-        'et': et,
-        'inf': inf,
-    }
+
+    # TODO Remove after testing.
+    time.sleep(3)
```

Comment:
Let's not forget to remove this. In testing, some of the jobs run over 5 seconds, so they timeout. After removing this, we can bump the `TaskModel.timeout` value up if the polling is still timing out.